### PR TITLE
Interface laws rhs

### PR DESCRIPTION
**Pull request by rbe051**
Fixes issue #196

### CODE REVIEW

**Comment by rbe051** [comment]
@alessiofumagalli I added you since I touched VEM :P My changes should not effect anything in your code, but changes the function headers. I added you to the review so you are aware about it.

@IvarStefansson This fixes the issue ?

**Comment by rbe051** [comment]
@keilegavlen no need to review much, but you ended up on review since you asked about the issue

**Comment by keileg** [comment]
Looks good to me.

File: examples/papers/arXiv_1809_06926/case1/data.py [hunk 1]
```diff
@@ -36,8 +36,16 @@ def add_data(gb, data, solver_name):
 
     is_fv = solver_name == "tpfa" or solver_name == "mpfa"
 
-    gb.add_node_props(["is_tangential", "problem", "frac_num", "low_zones",
```
[the remark this answers is not in the document]
Comment (alessiofumagalli) [reply examples/papers/arXiv_1809_06926/case1/data.py]: I normally run black only on the file I touch in the branch

**Comment by rbe051** [comment]
> Can you give a quick explanation what the branch add?

I updated the documentation for the changed functions. The branch adds the possibility to add a rhs to the coupling terms, e.g., assable_pressure_trace_int. So far this is needed in two places; to add the boundary condition contribution in the fv schemes, and the previous time step in the Biot equations.

**Comment by alessiofumagalli** [comment]
thanks @rbe051 , for me it's fine the merging

**Pull request merged**